tests/google/adwords.py

```diff
@@ -1,12 +1,15 @@
 import os
 from unittest import TestCase
 
-from googleads.adwords import ServiceQueryBuilder
+from googleads import adwords
+from pandas import DataFrame
+from pandas.util.testing import assert_frame_equal
 from py.builtin import execfile
 
 from hip_data_tools.google.adwords import GoogleAdWordsConnectionManager, \
```

Comment:
with these long imports is it better to import the module, so something like:
import hip_data_tools.google.adwords as adwords
then reference 
adwords.<CLASS>?
it would also mean you can drop the AdWords prefix to all the class names

Reply:
I let IntelliJ do its import optimisation. 
Either way is fine.